client/wc-api/settings/operations.js

```diff
@@ -62,7 +62,7 @@ function updateSettings( resourceNames, data, fetch ) {
 function settingsToSettingsResource( settings ) {
 	const settingsData = {};
 	settings.forEach( setting => ( settingsData[ setting.id ] = setting.value ) );
-	return { [ 'settings' ]: { data: settingsData } };
+	return { [ 'settings' ]: { data: settingsData, lastReceived: new Date() } };
```

Comment:
Updates here don't actually seem to get mapped to the store.  We need `lastReceived` to detect whether or not the update is still in progress and I would expect the error to get mapped to the store if the fetch in `updateSettings()` fails.

Any thoughts on how to handle this or did I approach this the wrong way? /cc @coderkevin

Reply:
`lastReceived` should automatically be set by `fresh-data` when you return the resources from the promise. I'm not sure what the problem is here. I'll contact you.